js/state/contentSettings.js

```diff
@@ -118,6 +118,10 @@ const getDefaultUserPrefContentSettings = (braveryDefaults, appSettings, appConf
       setting: getSetting(settings.METAMASK_PROMPT_DISMISSED) || getSetting(settings.METAMASK_ENABLED) ? 'block' : 'allow',
       primaryPattern: '*'
     }],
+    BATads: [{
+      setting: 'allow',
```

Comment:
this needs to be set to 'block' if the user hasn't opted into ad trials

Reply:
fixed